core/templates/pages/exploration-editor-page/modal-templates/exploration-modify-translations-modal.component.ts

```diff
@@ -164,4 +168,14 @@ export class ModifyTranslationsModalComponent extends ConfirmOrCancelModal {
       languageCode
     ) as string;
   }
+
+  isSetOfStringDataFormat(): boolean {
+    const activeDataFormat =
+      this.contentTranslations[Object.keys(this.contentTranslations)[0]]
```

Comment:
Please help me to understand why we are accessing the content translation of the first element only.

Reply:
It's because we want to determine the format of the content itself, and each element corresponds to a language. All elements are guaranteed to have the same format in the content translations dict regardless of language, so we just get it from the first element. Hope that makes sense!